controllers/comments.controllers.js

```diff
@@ -0,0 +1,22 @@
+const {
+  fetchCommentsByArticleId,
+  articleExists,
+} = require('../models/comments.models');
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+
+  articleExists(article_id)
+    .then((exists) => {
+      if (!exists) {
+        return res.status(404).send({ msg: 'Article not found' });
+      }
+      return fetchCommentsByArticleId(article_id);
+    })
+    .then((comments) => {
+      if (comments) {
+        res.status(200).send({ comments });
+      }
+    })
+    .catch(next);
```

Comment:
I like the use of a check exists function, but you should not be manually respond with the error like this. The check exists function show reject with an error and allow the catch to deal with it. Once this refactor has been done it also means we should be able to use a Promise.all(...) instead of a .then(...) chain as well.

We should also already have a model that tells us if an article id exists (HINT: from a previous task/endpoint)